libctru/include/3ds/svc.h

```diff
@@ -930,11 +988,19 @@ Result svcKernelSetState(u32 type, ...);
 void svcBreak(UserBreakType breakReason);
 
 /**
+ * @brief Breaks execution.
+ * @param breakReason Debug reason for breaking.
+ * @param debugParameter1 First debugging parameter
+ * @param debugParameter2 Second debugging parameter
+ */
+void svcDebugBreak(UserBreakType breakReason, u32 debugParameter1, u32 debugParameter2);
+
+/**
  * @brief Outputs a debug string.
  * @param str String to output.
- * @param length Length of the string to output.
+ * @param length Length of the string to output (needs to be positive).
  */
-Result svcOutputDebugString(const char* str, int length);
+Result svcOutputDebugString(const char* str, s32 length);
```

Comment:
I'm curious, what is the significance of a negative value when passed to this parameter?